src/hooks/useLiveSpecsExt.ts

```diff
@@ -120,7 +124,12 @@ export function useLiveSpecsExt_related(captureName: string) {
             .from(TABLES.LIVE_SPECS_EXT)
             .select(liveSpecsExtRelatedQuery)
             .eq('spec_type', 'materialization')
-            .or(`spec->>sourceCapture.eq.${captureName}`)
+            .or(ENABLED_SHARDS)
+            .or(
+                `spec->>sourceCapture.eq.${escapeReservedCharacters(
+                    captureName
+                )}`
+            )
```

Comment:
Added a check for the enabled shards and also escaping the `captureName` as I forgot to do this initially.